ci/eval/compare/default.nix

```diff
@@ -76,10 +76,14 @@ let
   getAttrs =
     dir:
     let
-      raw = builtins.readFile "${dir}/outpaths.json";
+      # Sorry.
+      raw = runCommand "outpaths.json" { } ''
+        zcat ${dir}/outpaths.json.gz > $out
+      '';
```

Comment:
I guess I'll comment on this now ~~before I wake up to an angry mob~~: the IFD is required for making the GHA's usage simpler while still keeping good out-of-GHA behavior.

Furthermore, I'd like to eventually migrate Hydra to use the reimplemented release checks (and a soon-to-come reimplemented `tarball` job), but all of those things more or less require the ability for us to pass eval results around without pulling in a comparatively giant closure (or without us losing the benefits of our current setup wrt parallelism between the system evals).

So if we have to make a singular consumer use IFD, I think it's an acceptable tradeoff.